Examples/DataRepresentation/Image/Image4.cxx

```diff
@@ -170,8 +170,7 @@ main(int, char *[])
 
   // Software Guide : BeginCodeSnippet
   // coordinates of the center of the first pixel in N-D
-  ImageType::PointType newOrigin;
-  newOrigin.Fill(0.0);
+  auto newOrigin = itk::MakeFilled<ImageType::PointType>(0.0);
```

Comment:
No problem, but I would prefer `{}` instead of `MakeFilled` for zero-initialization:

    ImageType::PointType newOrigin{};